src/NumberGenerator.js

```diff
@@ -0,0 +1,17 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
+class NumberGenerator {
+  createNumbersOfLotto = () => {
+    const numbersOfLotto = MissionUtils.Random.pickUniqueNumbersInRange(
+      1,
+      45,
+      6
+    );
+    numbersOfLotto.sort(function (a, b) {
+      return a - b;
+    });
```

Comment:
```suggestion
    numbersOfLotto.sort((a, b) => a - b);
```
arrow function을 사용하면 더 간단하게 나타낼 수 있습니다!